packages/lesswrong/lib/generated/fragmentTypes.d.ts

```diff
@@ -1660,6 +1659,14 @@ interface PetrovDayLaunch { // fragment on PetrovDayLaunchs
   readonly userId: string,
 }
 
+interface FeaturedResourcesDefaultFragment { // fragment on FeaturedResources
+  readonly title: string,
+  readonly body: string,
+  readonly ctaText: string,
+  readonly ctaUrl: string,
+  readonly expiresAt: Date,
+}
```

Comment:
@s-cheng you don't happen to remember why this got added, do you?

Reply:
No sorry, I assume I just ran `yarn generate` and this got added